src/components/com_tjcertificate/administrator/libraries/certificate.php

```diff
@@ -1135,4 +1175,26 @@ public function generateImageFromPDF($domPDFOutput)
 			}
 		}
 	}
+
+	/**
+	 * Get formated date
+	 *
+	 * @param   string  $datetime  The current offset
+	 *
+	 * @return  string  formatted datetime
+	 *
+	 * @since  __DEPLOY_VERSION__
+	 **/
+	public function getFormatedDate($datetime)
```

Comment:
You should use this function in the backend as well. Ignore if already using

Reply:
I will add while backend development

Reply:
what about this comment? @tushar33

Reply:
to use date format function in backend as well